subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/mutator/AbstractFileChangeMutator.groovy

```diff
@@ -16,18 +16,19 @@
 
 package org.gradle.performance.mutator
 
-import org.gradle.performance.fixture.BuildExperimentInvocationInfo
-import org.gradle.performance.fixture.BuildExperimentListener
-import org.gradle.performance.fixture.BuildExperimentListenerAdapter
-import org.gradle.performance.measure.MeasuredOperation
 
-abstract class AbstractFileChangeMutator extends BuildExperimentListenerAdapter {
+import org.gradle.profiler.BuildContext
+import org.gradle.profiler.BuildMutator
+
+abstract class AbstractFileChangeMutator implements BuildMutator {
```

Comment:
Can we remove this class and use `org.gradle.profiler.mutations.AbstractFileChangeMutator` instead?

Reply:
Didn't know about that, good to know.